lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Addresses.hs

```diff
@@ -356,7 +356,7 @@ spec = describe "SHELLEY_ADDRESSES" $ do
         verify r
             [ expectResponseCode HTTP.status200
             , expectField (Aeson.key "spending_key_hash_bech32" . Aeson._String)
-                (`shouldBe` "addr_test1hwl9tuz8uuqe8cnpv387d5kcj8gyz9r9q30x395vsvue5una0f4")
+                (`shouldBe` "addr_vkht1hwl9tuz8uuqe8cnpv387d5kcj8gyz9r9q30x395vsvue5una0f4")
```

Comment:
```
$ bech32 addr_vkh <<< addr_test1hwl9tuz8uuqe8cnpv387d5kcj8gyz9r9q30x395vsvue5una0f4
addr_vkh1hwl9tuz8uuqe8cnpv387d5kcj8gyz9r9q30x395vsvue5e44fh7
```

you cannot actually change the prefix of a bech32-encoded string "just like that" :sweat_smile:, the overall encoding depends on the prefix itself. A different prefix leads to a completely different string.

Reply:
Oh yes, thanks for fixing it... I started running this integration test, then forgot to come back to the test result and copy the correct encoding.